src/platform/common/globalActivation.node.ts

```diff
@@ -32,19 +31,14 @@ export class GlobalActivation implements IExtensionSingleActivationService {
         @inject(IConfigurationService) private configuration: IConfigurationService,
         @inject(IDocumentManager) private documentManager: IDocumentManager,
         @inject(IWorkspaceService) private workspace: IWorkspaceService,
-        @inject(CommandRegistry) private commandRegistry: CommandRegistry,
         @inject(IRawNotebookSupportedService) private rawSupported: IRawNotebookSupportedService
-    ) {
-        this.disposableRegistry.push(this.commandRegistry);
-    }
+    ) {}
 
     public get activationStartTime(): number {
         return this.startTime;
     }
 
     public async activate(): Promise<void> {
-        this.commandRegistry.register();
```

Comment:
Much better, a more uniform way of doing things.

Reply:
Yeah this is legacy stuff before we ever had IExtensionSingleActivationService. There's probably more places we'll find later.